kolibri/plugins/coach/assets/src/composables/useQuizCreation.js

```diff
@@ -117,19 +117,17 @@ export default function useQuizCreation() {
             },
             []
           );
-          if (removedResourceQuestionIds.length === 0) {
-            // If no resources were removed, we don't need to update the questions
-            return;
```

Comment:
This return here has making the whole function to return, and the updates werent applied when we had resource_pool updates.